resources/brew/brew.dsc.resource.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+export exist=true
+export NONINTERACTIVE=1
+
+check_args() {
+    if [[ -z $packageName ]]; then
+        echo "packageName not set"
+        exit 1
+    fi
+}
+
+to_json() {
+    while read line; do
+        echo $line | awk '{print "{ \"packageName\": \""$1"\", \"version\": \""$2"\", \"_exist\": "ENVIRON["exist"]" }"}'
+    done
+}
+
+if [ $# -eq 0 ]; then
+    echo "Command not provided, valid commands: get, export"
+    exit 1
+elif [[ $1 == "get" ]]; then
+    check_args
+    output="$(brew list ${packageName} --versions)"
+    if [[ $? -ne 0 ]]; then
+        export exist=false
+        output="${packageName}"
+    fi
+    echo $output | to_json
+elif [[ $1 == "set" ]]; then
+    check_args
+    if [[ -z $_exist ]]; then
+        _exist=true
```

Comment:
how would `$_exist` have otherwise been set?

Reply:
`dsc.exe` will set the environment to include `_exist` as an env var before this script is run.  That's how it passes the "JSON" payload as env var name-value pairs so there's no JSON parsing within the script.  Line 35-36 is so that `_exist` default to `true` if it's not specified in the original JSON payload.